packages/core/src/__tests__/transform.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+import { fromJS } from 'immutable';
+import { transform } from '..';
+
+describe('Transformers', () => {
```

Comment:
You are missing tests that make sure a transformer for a base kind (`['scene']`) will fire for a sub-kind (`['scene', 'new']`).